astropy/cosmology/parameters.py

```diff
@@ -250,9 +254,9 @@
                r"doi:  10.1086/377226. "
                r"Table 7 (WMAP + CBI + ACBAR + 2dFGRS + Lya)."
                r"\nPending WMAP team approval and subject to change.")
-)
+))
 
 
 # If new parameters are added, this list must be updated
-available = ['Planck18', 'Planck18_arXiv_v2', 'Planck15', 'Planck13',
-             'WMAP9', 'WMAP7', 'WMAP5', 'WMAP3', 'WMAP1']
+available = ('Planck18', 'Planck18_arXiv_v2', 'Planck15', 'Planck13',
```

Comment:
Maybe auto-generate? Any `MappingProxyType`?

Reply:
I want to offer more than just the ML parameters for each mission: the various other constrained fits, the full MCMC chains, etc.
So I'm going to move the parameter dicts to files at some point. Don't think autogen is necessary, given that.

Reply:
Sure, that will come then!